package.json

```diff
@@ -16,7 +16,7 @@
     "@babel/preset-typescript": "^7.17.12",
     "@react-native-community/async-storage": "^1.11.0",
     "@react-native-community/audio-toolkit": "git+https://github.com/tbenr/react-native-audio-toolkit.git#refs/tags/v2.0.3-status-v6",
-    "@react-native-community/blur": "^3.6.0",
+    "@react-native-community/blur": "git+https://github.com/Kureev/react-native-blur#refs/tags/v3.6.1",
```

Comment:
`v3.6.1` is required for fixing crash on android while navigating ([ref](https://github.com/Kureev/react-native-blur/pull/411)). But for some reason it only available on repo and not published on npm, even though it was  created on `Apr 27, 2021`